### PR TITLE
More listing and client tab fixes

### PR DESCRIPTION
**Pull request by tuxudo**
Localize more listings and tabs, fixed Munki listing, added encryption status to disk listing,
adjusting whitespace

### CODE REVIEW

**Comment by tuxudo** [comment]
There isn't much else to localize as far as I know 💃

File: app/views/listing/timemachine.php [hunk 1]
```diff
@@ -104,15 +104,16 @@
...
 	        	}
 	        	
-				// Format Check-In timestamp
-				var checkin = parseInt($('td:eq(7)', nRow).html());
-				var date = new Date(checkin * 1000);
-				$('td:eq(7)', nRow).html(moment(date).fromNow());
+	        	// Format Check-In timestamp
+	        	var checkin = parseInt($('td:eq(7)', nRow).html());
+	        	var date = new Date(checkin * 1000);
+	        	$('td:eq(7)', nRow).html('<span title="Checked in: ' + moment(date).format('llll') + '">'+moment(date).fromNow()+'</span>');
```
Comment (bochoven) [inline app/views/listing/timemachine.php]: "Checked in" needs localisation

**Comment by tuxudo** [comment]
Resolved merge conflict

File: app/modules/tag/views/tag_listing.php [hunk 2]
```diff
@@ -105,7 +105,7 @@
 			// Format Check-In timestamp
 			var checkin = parseInt($('td:last', nRow).html());
 			var date = new Date(checkin * 1000);
-			$('td:last', nRow).html(moment(date).fromNow());
+			$('td:last', nRow).html('<span title="Checked in: ' + moment(date).format('llll') + '">'+moment(date).fromNow()+'</span>');
```
Comment (tuxudo) [inline app/modules/tag/views/tag_listing.php]: Localized

File: app/views/listing/timemachine.php (hunk 1 above)
Comment (tuxudo) [inline app/views/listing/timemachine.php]: Localized

**Comment by tuxudo** [comment]
Added displays listing fix and resolved conflicts.

**Pull request merged**